Sccp_manager.inc/aminterface.class.php

```diff
@@ -447,39 +452,24 @@ function getSCCPVersion()
                         $result['vCode'] = 420;
                         break;
                     case 3. . .5:
+                        $result['vCode'] = 430;
                         if($version_parts[2] == 3){
                             $result['vCode'] = 433;
-                        } else {
-                            $result['vCode'] = 430;
                         }
                         break;
                     default:
                         $result['vCode'] = 400;
                         break;
                 }
             }
-            /* Revision got replaced by RevisionHash in 10404 (using the hash does not work) */
-            if (array_key_exists("Revision", $metadata)) {
-                if (base_convert($metadata["Revision"], 16, 10) == base_convert('702487a', 16, 10)) {
-                    $result['vCode'] = 431;
-                }
-                if (base_convert($metadata["Revision"], 16, 10) >= "10403") {
-                    $result['vCode'] = 431;
-                }
-            }
             if (array_key_exists("RevisionHash", $metadata)) {
                 $result['RevisionHash'] = $metadata["RevisionHash"];
-            } else {
-                $result['RevisionHash'] = '';
             }
             if (isset($metadata['RevisionNum'])) {
-                $result['RevisionNum'] = $metadata['RevisionNum'];
-                if ($metadata['RevisionNum'] >= 10403) { // new method, RevisionNum is incremental
-                    $result['vCode'] = 432;
-                }
-                if ($metadata['RevisionNum'] >= 10491) { // new method, RevisionNum is incremental
```

Comment:
I am happy to see we are leaving behind the old versions and concentrate only on last

Reply:
I think that before I commit, we should clone develop to say legacy, so that if someone cannot use the latest version of chan-sccp, there is a fallback
There are still bugs in PhantoVl even after PR108

Reply:
You can just set/add a tag that says "legacy" to save this moment in time.